Description: Add QtWidgets and QtGui explicitly as required Qt libraries.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/OpenBoard.pro
+++ b/OpenBoard.pro
@@ -39,6 +39,8 @@
 QT += webkitwidgets
 QT += multimediawidgets
 QT += printsupport
+QT += widgets
+QT += gui
 QT += core
 
 INCLUDEPATH += src
